components/climate/ir_climate.rst

```diff
@@ -154,19 +156,44 @@ Configuration variables:
         header_high: 3265us # AC Units from LG in Brazil, for example use these timings
         header_low: 9856us
 
+
+.. _climate_ir_panasonic:
+
+``Panasonic`` Climate
+---------------------------------
```

Comment:
the dashes length must match heading length

Reply:
My apologies, has been fixed